anyblok/conftest.py

```diff
@@ -0,0 +1,66 @@
+# This file is a part of the AnyBlok project
```

Comment:
I'm wondering if this file will be part of the released packages that we can import those feature in other packages ?

Reply:
it is my goal

Reply:
great ! thanks